README.md

```diff
@@ -49,6 +49,12 @@ Install the project dependencies:
 
     npm install
 
+Install Git hooks:
+
+    npm run setup-hooks
+
+*(This adds a pre-commit git hook which makes sure tests and passing and the code is eslint-compliant)*
```

Comment:
can you actually move this to CONTRIBUTING.md instead of the README?

I see the README instructions as more for people who want to use the project for other stuff, and the CONTRIBUTING.md for people who want to fork the repo and make their own changes (which is when they should add this git hook)